framework/doc/content/source/positions/MultiAppPositions.md

```diff
@@ -0,0 +1,25 @@
+# MultiAppPositions
+
+!syntax description /Positions/MultiAppPositions
+
+## Overview
+
+`MultiApps` have a position that is used to translate their frame of reference compared to
+the main app frame of reference. This position can be used to align the meshes for the transfers or
+for outputting the `MultiApps` [Exodus results](Exodus.md) at the translated positions.
```

Comment:
Does this leverage the coord transform object?

Reply:
I ll have to think about it. Depends on the order of operations I guess.

I would say no for the centroid case for sure